src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -90,8 +90,8 @@ public function addConfiguration(NodeBuilder $node): void
     {
         $node
             ->arrayNode($this->getKey())
-                ->fixXmlConfig($this->getKey())
                 ->fixXmlConfig('issuer')
+                ->fixXmlConfig('algorithm')
```

Comment:
This can be done in 7.3: while both keys `algorithm` and `algorithms`, they are coalesced in the end.